docs/dev/CodingGuidelines/CodingGuidelines.md

```diff
@@ -265,3 +266,42 @@ You should use the `async` / `await` pattern to avoid wrapping exceptions in `Ag
 ```C#
 public async Task TestDiagnostic(string actual) => await VerifyCSharpDiagnosticAsync(actual);
 ```
+
+### Async anonymous delegates 
```

Comment:
@k-s-popova please review the modified guide

Reply:
@SENya1990 , done